python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -442,7 +448,7 @@ def updateBlendRecords(blendData, catalog, modelPsf, observedPsf, redistributeIm
         contains those columns.
     """
     # We import here to avoid a circular dependency
```

Comment:
This seems like code smell...

Reply:
This is an artifact of the current state of the code and is not worth fixing. Before the scarlet models were exported, the only place to make measurements on the deconvolved models was in the `ScarletDeblendTask`. But these are band-dependent measurements and the output from `ScarletDeblendTask` is now a single monochromatic catalog. DM-34558 will create a sub-task to make these (and potentially other) measurements, but until that is created there is really no appropriate place for these measurements to live.